cmd/kots/cli/set-config.go

```diff
@@ -228,3 +234,19 @@ func getConfigValuesFromArgs(v *viper.Viper, args []string) ([]byte, error) {
 
 	return b, nil
 }
+
+func logConfigValidationErrors(groupValidationErrors []configtypes.ConfigGroupValidationError, log *logger.CLILogger) {
+	log.Errorf("Following config items have validation errors:")
+	for _, groupValidationError := range groupValidationErrors {
+		log.Errorf("Group - %s(%s)", groupValidationError.Title, groupValidationError.Name)
+		log.Errorf("  Items:")
+		for _, itemValidationError := range groupValidationError.ItemErrors {
+			log.Errorf("    Name: %s", itemValidationError.Name)
+			log.Errorf("    Type: %s", itemValidationError.Type)
+			log.Errorf("    Errors:")
+			for _, validationError := range itemValidationError.ValidationErrors {
+				log.Errorf("      - %s", validationError.Message)
+			}
+		}
+	}
+}
```

Comment:
Can you update the PR description to show an example of this?  Would help to review the UX

Reply:
added a screenshot
<img width="609" alt="image" src="https://user-images.githubusercontent.com/8703626/232098763-94f3b733-6cd0-4383-910d-b25c5deba27a.png">

Reply:
Would it be possible to not have all the dots at the beginning of each line? The dot indicates a new action and I feel like this entire error just belong to a single section.

Reply:
all the CLILogger functions start with a `dot`
Should I add a new logger function without `dot` or use `fmt.Print` to print them and wouldn;t be in red

Reply:
updated validation message for cli:
<img width="576" alt="image" src="https://user-images.githubusercontent.com/8703626/232512869-b551c5d6-9cf7-447f-acee-8f5156adf8f2.png">